docs/backup_workstations.rst

```diff
@@ -118,16 +115,90 @@ Then, copy the contents of the device's persistent volume to the directory using
 
 .. code:: sh
 
-  rsync -a --info=progress2 --no-specials --no-devices \
-      /media/amnesia/TailsData/ /media/amnesia/Backup/admin-backup
+  sudo bash -c "rsync -a --info=progress2 --no-specials --no-devices \
+      /media/amnesia/TailsData/ /media/amnesia/Backup/admin-backup && sync"
+
+.. note:: Note that the first path argument ends with a ``/``, but the second 
+          path argument should not!
 
-Once complete, unmount the TailsData partition.
+
+Once complete, unmount the TailsData partition by clicking the Eject button 
+beside its entry in the lefthand column of the file manager. When its entry is 
+no longer shown in the lefthand column, it is save to remove the 
+*Admin Workstation* USB.
 
 Repeat these steps for every device, making a new folder on the backup device
 for each device you back up.
 
 Finally, once you have completed the steps described in this section for each
-USB drive, unmount the Backup partition and store the drive somewhere safely.
+USB drive, unmount the Backup partition by clicking its Eject button. Wait until
+the Backup USB can be safely removed, and store it somewhere safely.
+
+.. note:: After the Eject button is clicked, it may be take some time before 
+          the drive can be safely removed. Wait until its entry  is removed from
+          the lefthand column of the file manager.
+
+Restoring a Workstation from a Backup
+=====================================
```

Comment:
This translates to a level 1 header, which means that it will be rendered similarly to a separate chapter both in the sidebar navigation and the table of contents:

![backup](https://user-images.githubusercontent.com/213636/57335800-a0879e00-70d8-11e9-862a-c9c4d328a739.png)

I think this is probably not what we want (as fas I can tell it's the only doc with more than one H1); instead I suggest we keep "Backup" as the H1 and create "Creating" and "Restoring" H2s.

Reply:
Done and done!